3_Multi channel display/Python_GUI_day3_file.py

```diff
@@ -2,98 +2,100 @@
 # numpy is used to read data from file
 import numpy as np
 
-global SERIAL_TAG1
-global SERIAL_TAG2
-global SERIAL_TAG3
+global tags
```

Comment:
As above, you can do directly:

```suggestion
TAGS = "series_tag1", "series_tag2", "series_tag3"
```
in capitals, if it is really a constant.